### DISCUSSION TITLE
How to inspect saved cookies

### DISCUSSION THREAD

**Discussion by jokester** [hoppscotch/hoppscotch#2784] [Q&A] [answered]
I'm using hoppsotch and Firefox extension in website development.

In a browser tab of the website, I don't see the cookies in Firefox's inspector saved. hoppsotch do send them to server though.

Can somebody give a hint about how the cookies storage work (and how to inspect them) ?

**Comment by AndrewBastin** [comment]
We don't have a clean and proper support for Cookie support right now due to current implementation being finnicky because of the browser restrictions.

We are working on making cookies a priority once the Hoppscotch Native project is released

**Comment by liyasthomas** [comment, marked as answer]
Cookies are now supported in Hoppscotch.
Documentation: https://docs.hoppscotch.io/documentation/features/cookies

Closing this ticket as this feature is now available in the recent version. Please feel free to reach out if you have any other concerns.
